test/test_helper.rb

```diff
@@ -9,3 +9,36 @@
 Dir["test/fixtures/**/*.rb"].each { |f| require File.expand_path(f) }
 
 Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+# Added assertions
+
+def assert_executes(exactly: nil, at_least: nil, message: "The given block did not execute the desired statement")
```

Comment:
Can we move this to its own module, so it can easily be included in the target project's `test_helper`?

Really, we should create `Slayer::Minitest`, and `Slayer::Rspec` modules that can be required into test helper to provide custom assertions.